java/wordcount-mapreduce/dp-mr-hb-init.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#    Copyright 2015 Google, Inc.
+# 
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+# 
+#        http://www.apache.org/licenses/LICENSE-2.0
+# 
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.set -x -e
+
+# Values you must set
+
+ROLE=$(/usr/share/google/get_metadata_value attributes/role)
```

Comment:
Switch this to attributes/dataproc-role like dataproc-bucket; we'll be removing the non-"dataproc"-prefixed but dataproc-specific metadata keys in the future.